### ISSUE TITLE
[Suggestion] ARM 32/64 support

### ISSUE THREAD

**Issue by tripplehelix** [music-assistant/companion#48]
Would be great to have ARM 32/64 bit builds, especially for Raspberry Pi devices.

**Comment by arctixdev** [comment]
Hi yes this would be good to have. I see that GitHub recently released arm runners for github actions which makes it a lot easier. Tauri has support for Arm so this should be as simple as adding it to the build pipeline. I dont have time to do this personally but anyone is welcome to give it a shot.

**Comment by arankwende** [comment]
This would be a great way to create Raspberry Pi screen companion for music assistant.

**Comment by arctixdev** [comment]
I agree, looks like it is super simple to implement, Tauri (the framework used to build the companion app) has full support for arm, so really the only thing needed is to modify the GitHub action that builds the companion and probably add a squeezelite executable for Linux arm

**Comment by troypsch** [comment]
Hi I wanted to send a note to see if this was implemented?
